src/deps.ts

```diff
@@ -40,13 +40,32 @@ export function toDependencies(object: {}): IDependency[]
  */
 export function parseVersion(version: string): string
 {
-  if (!/^[~^]/.test(version)) {
-    return version;
+  if (/^[~^]/.test(version)) {
+    return version.substring(1);
   }
 
-  return version.substring(1);
+  return extractVersionFromUrl(version);
 };
 
+/**
+ * @param  {string} version
+ * @returns string
+ */
+export function extractVersionFromUrl(version: string) : string
```

Comment:
Add comments to this method please, within the body!

Reply:
Happy to add comments, but there has been no precendent set for commenting in this project, hence I have not added any myself. I'll add them now.

Reply:
There are comments throughout various files, however smaller functions etc do not have tests...

- https://github.com/harksys/npmvet/blob/master/src/index.ts
- https://github.com/harksys/npmvet/blob/master/src/renderers/blessed.ts
- https://github.com/harksys/npmvet/blob/master/src/renderers/json.ts

etc etc.

Comments mostly for the more complex regex things and other developers trying to see what that regex does. :)